gallery/src/styles/NxTile/NxSimpleTileExample.html

```diff
@@ -6,10 +6,10 @@
     distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
 
 -->
-<div class="nx-tile">
-  <div class="nx-tile-header">
+<section class="nx-tile" aria-label="Example of nx-tile">
```

Comment:
Why doesn't the label match the header?

Reply:
> Why doesn't the label match the header?

I was trying to make the labels be more logical when read out loud, I should have had the word "simple" in there though. https://github.com/sonatype/sonatype-react-shared-components/pull/215/commits/92cc96f11ece11c47c9d259e9b2e4ccee8bca33f